frameworklauncher/src/main/java/com/microsoft/frameworklauncher/common/model/TaskStatus.java

```diff
@@ -131,6 +133,14 @@ public void setContainerIp(String containerIp) {
     this.containerIp = containerIp;
   }
 
+  public List<ValueRange> getContainerPorts() {
+    return containerPorts;
+  }
+
+  public void setContainerPorts(List<ValueRange> ports) {
+    containerPorts = ports;
+  }
```

Comment:
public void setContainerPorts(List<ValueRange> containerPorts ) {
    this.containerPorts = containerPorts ;
  }